langchain/langchain.config.js

```diff
@@ -65,6 +65,8 @@ export const config = {
     "chains/retrieval": "chains/retrieval",
     "chains/sql_db": "chains/sql_db/index",
     "chains/graph_qa/cypher": "chains/graph_qa/cypher",
+    // chat models
+    "chat_models": "chat_models/base",
```

Comment:
We talked about this before

Reply:
ah shit, sorry about that - blindly copying the py file path 🙃